src/components/StatusLabel/StatusLabel.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import type { HTMLProps, ReactNode } from "react";
+
+import classNames from "classnames";
+import { ClassName, PropsWithSpread, ValueOf } from "types";
+
+export const StatusLabelAppearance = {
+  Caution: "caution",
```

Comment:
Why the camel case here? it seems in other enums we use `ALLCAPS` (like in buttons)?

Reply:
Oh yeah sorry we've been moving to that syntax as it seems to be how it's written in the [TypeScript docs](https://www.typescriptlang.org/docs/handbook/enums.html). I'll change this to allcaps for consistency.